extensionsGallery-insider.json

```diff
@@ -3028,68 +3028,7 @@
 					],
 					"statistics": [],
 					"flags": ""
-				},
-				{
-					"extensionId": "63",
-					"extensionName": "arcdeployment",
-					"displayName": "Azure Arc deployment (deprecated)",
-					"shortDescription": "Provides a notebook-based experience to deploy resources on Azure Arc",
-					"publisher": {
-						"displayName": "Microsoft",
-						"publisherId": "Microsoft",
-						"publisherName": "Microsoft"
-					},
-					"versions": [
-						{
-							"version": "0.3.2",
-							"lastUpdated": "6/29/2020",
-							"assetUri": "",
-							"fallbackAssetUri": "fallbackAssetUri",
-							"files": [
-								{
-									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/arcdeployment/arcdeployment-0.3.2.vsix"
-								},
-								{
-									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/arcdeployment/content/extension.png"
-								},
-								{
-									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/arcdeployment/content/0.3.2/README.md"
-								},
-								{
-									"assetType": "Microsoft.VisualStudio.Code.Manifest",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/arcdeployment/content/0.3.2/package.json"
-								},
-								{
-									"assetType": "Microsoft.VisualStudio.Services.Content.License",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/arcdeployment/content/0.3.2/LICENSE.txt"
-								}
-							],
-							"properties": [
-								{
-									"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
-									"value": ""
-								},
-								{
-									"key": "Microsoft.AzDataEngine",
-									"value": ">=1.19.0"
-								},
-								{
-									"key": "Microsoft.VisualStudio.Code.Engine",
-									"value": ">=0.33.1"
-								},
-								{
-									"key": "Microsoft.VisualStudio.Services.Links.Source",
-									"value": "https://azure.microsoft.com/services/azure-arc/hybrid-data-services"
-								}
-							]
-						}
-					],
-					"statistics": [],
-					"flags": "preview"
-				},
+				}
 				{
 					"extensionId": "64",
```

Comment:
If the count is 64 and there's an extension id of 64, is that ok? No off by one errors, right?

Reply:
The count is just a count. If you look through the IDs we skip some (early ones that were removed) - so the count is not going to be the same as the highest ID.